client.py

```diff
@@ -297,6 +297,8 @@ async def on_ready():
 
         asyncio.ensure_future(loop_utils.decrease_food_multiplier())
 
+
+        asyncio.ensure_future(loop_utils.drunk_effects(id_server=server.id))
```

Comment:
Is this in front of pushdownServerInebriation? As in, like, does 100 inebriation correctly trigger its associated drunk_effect? Rhetorical question